### ISSUE TITLE
problem with init.tcl on Mac OS

### ISSUE THREAD

**Issue by mercuryin** [jlplenio/crude-twitch-viewer-bot#191]
**Please answer the following questions:**
- CTVBot version? [e.g. 0.4.0] Last one to date
- OS with version? [e.g. Windows 10, Ubuntu 20.04] Mac Ventura 
- Streaming platform? [e.g. twitch.tv, kick.com, youtube.com]
- Proxies from webshare.io? [yes, no] no

**Describe the bug**  

./CTVBot0.6.1_20230721_macOS_experimental.bin
Available proxies 1
Available window locations 2
Traceback (most recent call last):
  File "main_gui.py", line 24, in <module>
  File "ctvbot/gui.py", line 62, in __init__
  File "tkinter/__init__.py", line 2270, in __init__
_tkinter.TclError: Can't find a usable init.tcl in the following directories: 
    /usr/local/opt/tcl-tk/lib/tcl8.6 /usr/local/Cellar/tcl-tk/8.6.13_4/lib/tcl8.6 /Users/anonymous/.Trash/lib/tcl8.6 /Users/anonymous/lib/tcl8.6 /Users/anonymous/.Trash/library /Users/anonymous/library /Users/anonymous/tcl8.6.13/library /Users/tcl8.6.13/library

This probably means that Tcl wasn't installed properly.

**Screenshots**  

**Logs**  
If applicable, post the part of the ctvbot.log, that refers to the problem.

**Comment by mercuryin** [comment]
I have just found others with same problem and you said is not an easy fix and you will look into it so for now I am just using my pc instead thanks anyways

**Comment by jlplenio** [comment]
Thank you for finding the answer on your own. Was it the problem with Apple silicon?

**Comment by mercuryin** [comment]
I'm sorry if I explained myself poorly. What I meant to say is that I couldn't solve the problem because it seems complicated. After posting my question, I found someone else asking the same thing, and you replied that you didn't have a solution at the moment. So, I took the easy route and installed it on Windows, and it worked perfectly from the start. I'm sorry that I couldn't provide any help. I tried for an entire afternoon to get it running on my MacBook, but it didn't seem possible, so it looks like a complex issue. If you don't mind, I have a question regarding your proxy guide. You recommend datacenter proxies over residential, probably because they are more cost-effective. Is that correct, or did I misunderstand? Thank you.
